src/uk/ac/cam/cl/interaction_design/group19/app/Icons.java

```diff
@@ -0,0 +1,153 @@
+package uk.ac.cam.cl.interaction_design.group19.app;
+
+import java.awt.Dimension;
+import java.awt.Graphics2D;
+import java.awt.Image;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import javax.imageio.ImageIO;
+
+public class Icons {
+
+    public static BufferedImage getSizedIcon(WeatherType theWeather, int width) {
+        BufferedImage temp = getIcon(theWeather);
+        Image toRet = temp.getScaledInstance(width, width * temp.getHeight() / temp.getWidth(), Image.SCALE_SMOOTH);
+        return toBufferedImage(toRet);
+    }
+
+    public static BufferedImage getIcon(WeatherType theWeather) {
+        String icon = "question.png";
```

Comment:
I think you should use a hashmap to cache the icons so they don't have to be read in multiple times